src/phoenix/datasets/dataset.py

```diff
@@ -324,6 +367,23 @@ def _check_embedding_column_names_for_excluded_columns(
             unseen_column_names.discard(column_name)
 
 
+def _check_relationship_column_names_for_excluded_columns(
```

Comment:
this function is not used anywhere

Reply:
Yeah it's not - let me remove it - the weird thing is that we import stuff from this file so without this i might get an import error. I can refactor the tests though.